### PR TITLE
Assigning numbers via powershell

### PR DESCRIPTION
**Pull request by cchiffersMSFT**
Adding in some details about assigning numbers via powershell.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md [hunk 1]
```diff
@@ -208,6 +208,31 @@ See [Set auto attendant languages for Audio Conferencing](set-auto-attendant-lan
 
   
 See [See a list of Audio Conferencing numbers](see-a-list-of-audio-conferencing-numbers-in-teams.md).
+
+## Setting audio conferencing dial-in numbers in PowerShell
+
+You can set a users audio conferencing toll and toll free number in PowerShell.
+
+1. Connect to the Microsoft Teams service
```
Comment (JohanFreelancer9) [inline Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md:216]:
```suggestion
You can set a user's audio conferencing toll and toll-free number in PowerShell.

1. Connect to the Microsoft Teams service.
```

File: Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md [hunk 2]
```diff
@@ -208,6 +208,31 @@ See [Set auto attendant languages for Audio Conferencing](set-auto-attendant-lan
...
+
+```powershell
+PS C:\> Connect-MicrosoftTeams
+```
+2. Assign a toll number to a user (Example: Assign the toll number +612 800 0000 to sarah@contoso.com)
```
Comment (JohanFreelancer9) [inline Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md:221]:
```suggestion
2. Assign a toll number to a user (for example, assign the toll number +612 800 0000 to sarah@contoso.com).
```

File: Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md [hunk 3]
```diff
@@ -208,6 +208,31 @@ See [Set auto attendant languages for Audio Conferencing](set-auto-attendant-lan
...
+
+```powershell
+PS C:\> Set-CsOnlineDialInConferencingUser sarah@contoso.com -ServiceNumber 6128000000
+```
+
+3. Optional: Assign a toll free number to a user (Example: Assign the toll free number +61 1800 000 000 to sarah@contoso.com)
```
Comment (JohanFreelancer9) [inline Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md:227]:
```suggestion
3. Optional: assign a toll-free number to a user (for example, assign the toll-free number +61 1800 000 000 to sarah@contoso.com).
```

File: Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md [hunk 4]
```diff
@@ -208,6 +208,31 @@ See [Set auto attendant languages for Audio Conferencing](set-auto-attendant-lan
...
+
+```powershell
+PS C:\> Set-CsOnlineDialInConferencingUser sarah@contoso.com -ServiceNumber 6128001234
+```
+
+[!NOTE]
+> You can only assign toll or toll free numbers that are registered within your tenant. 
+
```
Comment (JohanFreelancer9) [inline Teams/manage-the-audio-conferencing-settings-for-my-organization-in-teams.md:235]:
```suggestion
> You can only assign toll or toll-free numbers that are registered within your tenant. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cchiffersMSFT, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cchiffersMSFT** [comment]
Looks like there's some conflicts. I'm attempting to resolve now.

**Comment by cchiffersMSFT** [comment]
@scanum @tonysmit - conflicts resolved.

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**